### ISSUE TITLE
drqa - train log no update

### ISSUE THREAD

**Issue by avr248** [facebookresearch/ParlAI#470]
[ time:28s parleys:740 time_left:33654s num_epochs:0 ] {'total': 54, 'accuracy': 0, 'f1': 0, 'hits@k': {1: 0, 5: 0, 10: 0, 100: 0}}

**Comment by alexholdenmiller** [comment]
Hi @avr248, could you provide more information (the full command you ran, what you were expecting to see)?

**Comment by avr248** [comment]
command:
python examples/train_model.py -m drqa -t squad -mf 'out/drqa_squad'  --embedding_file .....

**i want to see like this:**
[ time:6s parleys:828 time_left:648s num_epochs:0 ] {'total': 273, 'accuracy': 0.1465, 'f1': 0.1587, 'hits@k': {1: 0.147, 5: 0.147, 10: 0.147, 100: 0.147}, 'loss': 2.42}

**Comment by alexholdenmiller** [comment]
The DrQA model chooses not to return predictions during training (which can speed up training when using a GPU, as that requires copying predictions back from the GPU to the host). This prevents those metrics such as accuracy and f1 from being populated in the train logs.

If you want to see any more than the train_loss during training, you'll have to customize the model to return more data, or just set the `vtim` flag to run validation during training (e.g. `-vtim 300` will run validation every 5 minutes).

**Comment by avr248** [comment]
thanks
